server/config/config.go

```diff
@@ -185,6 +209,14 @@ func DefaultConfig() *Config {
 			Enable:  true,
 			Address: DefaultGRPCAddress,
 		},
+		Rosetta: RosettaConfig{
+			Enable:     false,
+			Address:    ":8080",
```

Comment:
Why not using `vyper` here?

Reply:
These are the defaults, below we take the vyper ones.